cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ServiceConfiguration.java

```diff
@@ -535,6 +642,28 @@ public Builder jvmOptions(Collection<String> jvmOptions) {
             return this;
         }
 
+        /**
+         * The jvm options for the new service. They will be added directly before the "-Xmx" parameter in the startup command.
+         */
+        public Builder jvmOptions(String... jvmOptions) {
+            return this.jvmOptions(Arrays.asList(jvmOptions));
+        }
+
+        /**
+         * The jvm options for the new service. They will be added directly before the "-Xmx" parameter in the startup command.
+         */
+        public Builder addJvmOptions(String... jvmOptions) {
+            return this.addJvmOptions(Arrays.asList(jvmOptions));
+        }
+
+        /**
+         * The jvm options for the new service. They will be added directly before the "-Xmx" parameter in the startup command.
+         */
+        public Builder addJvmOptions(Collection<String> jvmOptions) {
+            this.config.processConfig.jvmOptions.addAll(jvmOptions);
```

Comment:
I think the jvmOptions could also be null here

Reply:
Same as above